webapp/src/gesture/installMlp.ts

```
@@ -259,38 +298,117 @@ export function installMlp() {
       return false;
     }
+    
+    // Try custom model data first (for profile models)
+    if (customModelData) {
+      const modelData = customModelData as string;
+      if (await loadMlpFromB64(modelData)) {
+        forwardTelemetry('mlp_custom_loaded', { size: modelData.length });
+        return true;
+      }
+      throw new Error('Failed to load custom model data');
+    }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)